srcs/app/CGI/CGI.cpp

```diff
@@ -38,6 +38,7 @@ std::map<std::string, std::string> CGI::MakeEnv_(void) {
 	env_.insert(std::make_pair("REDIRECT_STATUS", "200"));
```

Comment:
what about [PATH_INFO](https://www.rfc-editor.org/rfc/rfc3875#section-4.1.5.)

from the subject:

> Because you won’t call the CGI directly use the full path as PATH_INFO

Reply:
Good catch! Thanks for the image with all the environment variables.

The `PATH_INFO` is a bit more complex (and the description from the subject is incorrect), needs more research.

For example, according to [RFC 3875](https://datatracker.ietf.org/doc/html/rfc3875#section-4.1.5), for a request to this address:
```
 http://example.com/cgi-bin/somescript.cgi/this%2eis%2epath%3binfo
```

`SCRIPT_NAME` is `/cgi-bin/somescript.cgi`
`PATH_INFO` is `/this.is.the.path;info`

I've created a new issue to track this, https://github.com/42AGV/webserv/issues/135

Reply:
#143